PWGJE/EMCALJetTasks/macros/AddTaskRecoilJetYield.C

```diff
@@ -21,10 +21,10 @@ AliAnalysisTaskRecoilJetYield* AddTaskRecoilJetYield(const char * njetsData, //d
 								    TString     trigClass      = "",
 								    TString     kEmcalTriggers = "",
 								    TString     tag            = "",
-								    AliAnalysisTaskRecoilJetYield::JetShapeType jetShapeType,
-								    AliAnalysisTaskRecoilJetYield::JetShapeSub jetShapeSub,
-								    AliAnalysisTaskRecoilJetYield::JetSelectionType jetSelection,
-								    Float_t minpTHTrigger =0.,  Float_t maxpTHTrigger =0., AliAnalysisTaskRecoilJetYield::DerivSubtrOrder derivSubtrOrder = 0  ) {
+								    AliAnalysisTaskRecoilJetYield::JetShapeType jetShapeType = 0,
+								    AliAnalysisTaskRecoilJetYield::JetShapeSub jetShapeSub = 0,
+								    AliAnalysisTaskRecoilJetYield::JetSelectionType jetSelection = 0,
```

Comment:
Here ROOT6 might still complain about a type mismatch for the default argument (variable type is AliAnalysisTaskRecoilJetYield::JetSelectionType) - please use the proper enum value.